File: app/src/app/dto/client-payment.ts
export class ClientPayment{
    public amount:number;
    public date:string;
    public fullName:string;
    public idPayment:number;
    public identification:string;
    public pin:string;
    constructor() {
        this.amount=0;
        this.date='';
        this.fullName='';
        this.idPayment=0;
        this.identification='';
        this.pin = '';
    }
}

File: api/Clients/Domain/DTO/ClientPayment.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace api.Clientes.Domain.DTO
{
    public class ClientPayment
    {
        [Key]
        public int IdPayment { get; set; }
        public string FullName { get; set; }
        public string Identification { get; set; }
        public string PIN { get; set; }
        public decimal Amount { get; set; }
        public DateTime Date { get; set; }
    }
}
